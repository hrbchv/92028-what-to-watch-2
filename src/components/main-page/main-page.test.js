import React from "react";
import MainPage from "./main-page.jsx";
import Adapter from "enzyme-adapter-react-16/build/index";
import Enzyme, {shallow} from "enzyme/build/index";

Enzyme.configure({adapter: new Adapter()});

describe(`MainPage`, () => {
  it(`should renders correctly`, () => {
    const tree = shallow(<MainPage/>);
    expect(tree).toMatchSnapshot();
  });
});
