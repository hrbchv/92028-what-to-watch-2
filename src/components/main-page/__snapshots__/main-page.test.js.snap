// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage should renders correctly 1`] = `ShallowWrapper {}`;
